paradise.dme

```diff
@@ -1822,6 +1822,7 @@
 #include "code\modules\shuttles\shuttle_supply.dm"
 #include "code\modules\shuttles\shuttles_multi.dm"
 #include "code\modules\shuttles\whiteship.dm"
+#include "code\modules\space transition\space_transition.dm"
```

Comment:
Interesting then that /tg/ let it slide if it's such a major issue, and that we've functioned for two years without ever having to get rid the spaces or running into any issues with them. Relative code looks like shit and is hard to work with: this is not. Not to mention that the standard in the modules file is actually spaces rather than underscores.

But very well.